### ISSUE TITLE
In version 1.16, the default lalt+W cannot remove the black border.

### ISSUE THREAD

**Issue by Helaer** [Genymobile/scrcpy#1690]
By default, lalt+W cannot remove the black borders,Some of my computers have no response. Some computers are only useful after restarting, and some computers are always unresponsive.

**Comment by Helaer** [comment]
My computer system is windows.

**Comment by rom1v** [comment]
If you resize arbitrarily the window (so that black borders are added), then press <kbd>Alt</kbd>+<kbd>w</kbd>, it does nothing? And with <kbd>Windows</kbd>+<kbd>w</kbd>?

**Comment by Helaer** [comment]
Yes, after scrcpy is started by default, for some of my windows computers, pressing lalt+w does nothing and cannot remove the black border.

**Comment by rom1v** [comment]
> after scrcpy is started by default

By default, scrcpy is started at optimal size, so there are not black borders added around the screen. Please post a screenshot.

**Comment by Helaer** [comment]
I tested it when the black area appeared when dragging the border.

**Comment by Helaer** [comment]
I mean when there is a black border, lalt+w does not remove the black border.

**Comment by rom1v** [comment]
Please post a screenshot of the scrcpy window when the windows is not at its optimal size and lalt+w does not resize it to its optimal size (removing black borders around the devixe screen).
